apps/services/user-profile/src/app/worker/userProfileAdvania.model.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  Column,
+  DataType,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+
+import { ProcessedStatus } from './types'
+
+@Table({
+  tableName: 'user_profile_advania',
+  timestamps: true,
+  indexes: [
+    {
+      fields: ['ssn'],
+    },
+  ],
+})
+export class UserProfileAdvania extends Model {
+  @CreatedAt
+  @ApiProperty()
```

Comment:
~ `@ApiProperty` is not needed for this model as we are not returning it in a rest endpoint. But it doesn't harm anything so up to you to remove it or leave it.